documentation/API-design-guidelines.md

```diff
@@ -1389,18 +1403,19 @@ curl -X 'POST' \
  ```
  ```json 
 {
-  "eventSubscriptionId": "456g899g",
-  "event": {
-    "eventType": "SUBSCRIPTION_ENDS",
-    "eventTime": "2023-01-24T13:18:23.682Z",
-    "eventDetail": {
-      "ueId": {
-        "ipv4Addr": "192.168.0.1"
-      },
-      "uePort": 5060,
-      "terminationReason": "Service terminates for lack of consent"
-    }
-  }
+  "id": 123658,
+  "source": "https://notificationSendServer12.supertelco.com",
+  "type": "org.camara.api.device-status.SUBSCRIPTION_ENDS",
+  "specversion": "1.0",
+  "datacontenttype": "application/json",
+  "data": {
+    "eventSubscriptionId": "456g899g",
+    "device": {
+      "phoneNumber": 123456789
+    },
+    "terminationReason": "SUBSCRIPTION_EXPIRED"
+  },
+  "time": "2023-01-19T13:18:23.682Z"
 }
 ```
```

Comment:
A section in the end on Security where we can point to https://github.com/cloudevents/spec/blob/v1.0.2/cloudevents/spec.md#privacy-and-security

and also point that producers shall implement mechanisms at their end to ensure Abuse protection https://github.com/cloudevents/spec/blob/main/cloudevents/http-webhook.md#4-abuse-protection

Reply:
@shilpa-padgaonkar Added 2 paragraphs to mention these point and link to CloudEvents specification.